otx/algorithms/common/configs/configuration_enums.py

```diff
@@ -33,3 +33,16 @@ class StorageCacheScheme(ConfigurableEnum):
     JPEG_95 = "JPEG/95"
     PNG = "PNG"
     TIFF = "TIFF"
+
+
+class BatchSizeAdaptType(ConfigurableEnum):
+    """This Enum represents the type of adapting batch size.
+
+    None : Not adapt batch size.
+    Safe : Find a batch size preventing GPU out of memory.
```

Comment:
I'm still a little confused, Safe is the decrease_only mode, right?
Sentence "Find a batch size preventing GPU out of memory" seems like it should apply to both Safe and Full.

Reply:
Do you mean "Find a batch size preventing GPU out of memory" needs to be added to `Full` description?